.github/workflows/get-values.yaml

```
@@ -0,0 +1,59 @@
+          ${{ toJSON(github) }}
+          JSON
+
+      - name: Checkout code
+        uses: actions/checkout@v5.0.0
+        with:
+          persist-credentials: false
+
+      - name: Update Devcontainer Hash
+        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'push' }}
+        id: update-hash
+        uses: ./.github/actions/update-devcontainer-hash
```

actionlint (1.7.9)

description is required in metadata of "Update Devcontainer Hash" action at "/home/jailuser/git/.github/actions/update-devcontainer-hash/action.yml"

(action)